docs/Database.md

```diff
@@ -0,0 +1,10 @@
+# Database
+*
+Databasename: question_library
+*				
+Tablename: questions
+*
```

Comment:
No asterisks for empty lines; use asterisks to enumerate something. If you want two lines to be displayed as two, just use a newline between them.